### ISSUE TITLE
Rework the installation logic

### ISSUE THREAD

**Issue by samilliken** [samilliken/openDCIM#1191]
Currently the logic requires that the install.php file be removed - when installing via containers, that won't work. 

We do still need a mechanism to ensure that only one session is running the install at a time, which was the original intent of our poor man's deadlock.

It will be much simpler if we draw a line stating that going forward, we will only upgrade directly from 18.01 or newer.    If you are running an older version than that, upgrade to either 18.01, 18.02, or 19.01, first.

**Comment by timlegge** [comment]
I just confirmed that going to login_ldap.php with SAML enabled and entering dcim and the new LDAP emergency password allows you access.

I had thought of what I might need to do to enable that functionality but that seems fine as a work around should saml be broken.

I suppose I could add a saml/no_sso.php page and have it do a redirect to ldap_login.php.

Also, adding/enabling a logout button when the user logs in with the emergency password seems like a good idea.  

I simply went to the saml login page directly to force the login but some users will likely get stuck on how to logout to test the settings.

Tim
